distribution/docs/src/main/resources/_contents/_developing/contributing-intro-contents.adoc

```diff
@@ -0,0 +1,5 @@
+
+The ${ddf-branding-expanded} is free and open-source software offered under the GNU Lesser General Public License.
+The Distributed Data Framework is managed under the guidance of the http://codice.org[Codice Foundation].
```

Comment:
Yes, or possibly just ${ddf-branding} as it's spelled out in the previous sentence.